src/Octoshift/Services/MigrateRepoErrorHandler.cs

```diff
@@ -0,0 +1,18 @@
+using System.Diagnostics.Contracts;
+
+namespace OctoshiftCLI.Services;
+
+public static class MigrateRepoErrorHandler
+{
+    public static OctoshiftCliException DecorateInsufficientPermissionsException(OctoshiftCliException ex, string orgName)
+    {
+        Contract.Requires(ex != null);
+        var errorSuffix = GenerateInsufficientPermissionsHelpMessage(orgName);
+        return new OctoshiftCliException($"{ex.Message}. {errorSuffix}", ex);
```

Comment:
```suggestion
        var errorSuffix = GenerateInsufficientPermissionsHelpMessage(orgName);
        return new OctoshiftCliException($"{ex?.Message}. {errorSuffix}", ex);
```

note the ? in ex?.Message. that should get rid of the formatter error.